api-reference/10 UI Widgets/dxTreeView/1 Configuration/createChildren.md

```diff
@@ -81,6 +81,69 @@ The following code shows how to use this function with a remote service:
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxTreeView
+            :create-children="createChildren"
+            :root-value="''"
+            data-structure="plain"
+            />
+        </div>
+    </template>
+    <script>
+
+    import DxTreeView from 'devextreme-vue/tree-view';
+    import 'whatwg-fetch';
+
+    export default {
+        components: {
+            DxTreeView
+        },
+        methods: {
+            createChildren: function(parent) {
+            let parentId = parent ? parent.itemData.id : '';
+
+            return fetch(`https://js.devexpress.com/Demos/Mvc/api/TreeViewData?parentId=${parentId}`)
```

Comment:
It;s a good idea to use a real example but it's inconsistent with angular and jQ approaches. Please make them consistent if you would like to use this URL.

Reply:
Fixed  - https://github.com/DevExpress/devextreme-documentation/pull/1555/commits/27e99f72319e69d41644da6d98ce34d858fe67ba